### ISSUE TITLE
about thermal pressurization

### ISSUE THREAD

**Issue by yayadian2012** [ydluo/qdyn#20]
Dear QDYN developers

Is there any scaling relations between the parameter w (half-width of shear distribution [m]) and the length of element, or any requirement should be meet? All of the thermal pressurization parameters should be set for the whole model or only part of the model?  

Looking forward for your reply! Thank you!

**Comment by martijnende** [comment]
No, there is no relation between half-width and element size. The half-width merely represents the degree of localisation of shear strain. Each of the parameters are set for individual elements, same as the other friction parameters. Note that the thermal pressurisation module has not been tested to a satisfactory degree, so use it at your own risk.

**Comment by yayadian2012** [comment]
Thans for your reply.
Are you plan to improve the thermal pressurization module and give an example? I'll be waiting for your good news.
